JS/index.js

```diff
@@ -0,0 +1,14 @@
+$("document").ready(function() {
+
+console.log("test");
```

Comment:
Remove console logs before you commit. This takes up file space and no longer provides a use.